Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -0,0 +1,2194 @@
+      isArray: false
+      name: issueId
+      required: false
+      hidden: true
+      secret: false
+    - default: false
+      description: The issue ID (Issue ID or key is required).
+      isArray: false
+      name: issue_id
+      required: false
+      secret: false
+    - default: false
+      description: The issue key (Issue ID or key is required).
+      isArray: false
+      name: issue_key
+      required: false
+      secret: false
+    - default: false
+      description: The application type of the linked remote application. For example "com.atlassian.confluence".
+      isArray: false
+      name: application_type
+      required: false
+      secret: false
+    - default: false
+      description: Please use application_type.
+      isArray: false
+      name: applicationType
+      required: false
+      hidden: true
+      secret: false
+    - default: false
+      description: The application name of the linked remote application. For example "My
+        Confluence Instance".
+      isArray: false
+      name: application_name
+      required: false
+      secret: false
+    - default: false
+      description: Please use application_name.
+      isArray: false
+      name: applicationName
+      required: false
+      hidden: true
+      secret: false
+    deprecated: false
+    description: Creates (or updates) an issue link.
+    execution: false
+    name: jira-issue-add-link
+  - arguments:
+    - default: false
+      description: The user attribute value. For Jira Cloud, this value can be the display name, or email address,
+        and for Jira OnPrem, this value can be the display name, email address, or username.
+      isArray: false
+      name: attribute
+      required: true
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: The maximum number of users to fetch when searching for a matching
+        user (default is 50). The maximum allowed value is dictated by the Jira property
+        'jira.search.views.default.max'. If you specify a value greater than
+        this number, your search results are truncated.
+      isArray: false
+      name: max_results
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets the account ID for a given user attribute.
+    execution: false
+    name: jira-get-id-by-attribute
+    outputs:
+    - contextPath: Jira.User.Attribute
+      description: The user's attribute.
+      type: String
+    - contextPath: Jira.User.AccountId
+      description: The user's account ID.
+      type: String
+  - arguments:
+    - default: true
+      description: The JQL query string.
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: The index (integer) of the first issue to return (0-based).
+      isArray: false
+      name: start_at
+      required: false
+      secret: false
+    - default: false
+      description: Please ue start_at.
```

Comment:
These arguments were kept in the YML file so we keep the integration backwards compatible, since this is a V3, and a lot of users are already use the V2 integration, we decided to keep the arguments that were used in V2, and mark them as **hidden: true**, this way, we don't break backwards compatibility.